### ISSUE TITLE
Matomo 3.8.1 update?

### ISSUE THREAD

**Issue by lboynton** [matomo-org/matomo-package#94]
Hi

Will there be an update to the debian repo at http://debian.piwik.org to include 3.8.1 soon?

Thanks

**Comment by anoma** [comment]
I would like to ask about release schedules. Please, I understand that it is very hard to ask this question without sounding disingenuous, but I really do just want to understand how to plan windows for upgrades.

As of today Feb 03, 2019 matomo-package is at 3.7.0 which was released upstream November 19, 2018.

Since 3.7.0 upstream have released 3.8.0 on January 21, 2019 and 3.8.1 on January 28, 2019.

Matomo (as shipped here) is feeding back that 3.8.x versions are available upgrades to end users even though in real terms they are not. As an aside I would suggest a valid refinement would be to disable this confusing alert by default or alter the words to explain clearer what it means.

However the real problem I have is that 3.8.0 is a "Security release - This release is rated critical". 

Expectations can be managed and plans can be made but it is hard to do so without some sort of standard delay to plan around.

Whats the current thinking around release schedules?

**Comment by jawrat** [comment]
@anoma there's always the option of installing it manually if you want to always have the latest updates.  iirc, the matomo updater works well if you're running it on a single server.  one of the reasons I went to the packages in the first place was that it was easier to upgrade multiple servers (since I couldn't do it from the front end)

have in mind that the package maintainer(s) are people too and do this in their spare time for basically nothing.  at the current time, there's an issue with the changelog (see other issue here in github) that has yet to be resolved, but i'm sure will be at some point.  until then, 3.7.0 will just have to do... :)

**Comment by anoma** [comment]
I have no issues with that but it doesn't match up with the official site:

https://matomo.org/faq/how-to-install/#faq_17844
```
How do I install Matomo on Debian GNU/Linux servers?

Matomo (Piwik) team maintains a package and a repository for Debian.
It lets you install Matomo in seconds using sudo apt-get install piwik.
To get started and be able to run this command, read the Matomo Debian package blog post.
```

There are worlds of difference between the official statement of  `Matomo (Piwik) team maintains a package` and volunteers ` do this in their spare time`.

**Comment by jawrat** [comment]
@anoma fair enough.  so either the docs need to be modified, or the matomo team needs to step up and pay someone to maintain the repo.  if I had the bandwidth personally, i'd get involved myself, but i've no experience running/maintaining a repo nor have I time to dig in to it at the moment. so, I wait.  @aureq will get to it when he can i'm sure.  :)

**Comment by lboynton** [comment]
Thank you @aureq!

**Comment by mattab** [comment]
fyi our plan is to eventually automate the process but we haven't yet got to it - hopefully this year :_)

**Comment by anoma** [comment]
@mattab thanks that one line helps me a lot. Keep up the good work everyone, always know it is appreciated.

**Comment by aureq** [comment]
`3.9.0` has been released and show follow soon.
